scripts/cat/cats.py

```diff
@@ -1200,101 +1162,60 @@ def generate_lead_ceremony(self):
         )
 
         # if we have relations, then make sure we only take the top 8
-        if dead_relations:
-            i = 0
-            for rel in dead_relations:
-                if i == 8:
-                    break
-                if rel.cat_to.status == "leader":
-                    life_giving_leader = rel.cat_to
-                    continue
-                life_givers.append(rel.cat_to.ID)
-                i += 1
+        for i in range(min(8,len(dead_relations))):
+            if rel.cat_to.status == "leader":
+                life_giving_leader = rel.cat_to
+                continue
+            life_givers.append(rel.cat_to.ID)
+
         # check amount of life givers, if we need more, then grab from the other dead cats
         if len(life_givers) < 8:
             amount = 8 - len(life_givers)
 
             if starclan:
-                # this part just checks how many SC cats are available, if there aren't enough to fill all the slots,
-                # then we just take however many are available
-
-                possible_sc_cats = [
-                    i
-                    for i in game.clan.starclan_cats
-                    if self.fetch_cat(i)
-                    and i not in life_givers
-                    and self.fetch_cat(i).status not in ["leader", "newborn"]
-                ]
+                cats_in_starcaln_or_darkforest = game.clan.starclan_cats
```

Comment:
```suggestion
                cats_in_starclan_or_darkforest = game.clan.starclan_cats
```
Please cascade this change through all uses of this variable.

Reply:
Resolved, typo fixed